lib/rollbar/capistrano3.rb

```diff
@@ -7,24 +7,27 @@
 require 'rollbar/capistrano_tasks'
 
 namespace :rollbar do
+  # dry_run? wasn't introduced till Capistrano 3.5.0; use the old fetch(:sshkit_backed)
+  set :dry_run, (proc { ::Capistrano::Configuration.env.fetch(:sshkit_backend) == ::SSHKit::Backend::Printer })
+
   desc 'Send deployment started notification to Rollbar.'
   task :deploy_started do
     on primary fetch(:rollbar_role) do
-      ::Rollbar::CapistranoTasks.deploy_started(self, self, false)
+      ::Rollbar::CapistranoTasks.deploy_started(self, self, fetch(:dry_run))
```

Comment:
The previous call had been `dry_run?`, and here you are not using the `?`. Maybe that is why the other one was failing? Otherwise, does this need to have the `?` as well?

Reply:
No. Previously this was failing for a user on Capistrano 3.4 which didn't have the `dry_run?` helper method. It was introduced in Capistrano 3.5. Because of this I need to manually verify the dry run using the same logic that was used before 3.5 and set it in `dry_run` property which is what I'm doing on line 11. Then I `fetch` that property.

Does it make sense?

Reply:
Yes, and I overlooked line 11. Thank you for the explanation!